src/styles/main.scss

```diff
@@ -1,3 +1,55 @@
+$color-hover: #e3b2b2;
+$cell-width: 100px;
+$cell-height: 100px;
+$padding-calendar: 10px;
+$cell-gap: 1px;
+$week-days: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: #{$padding-calendar};
+  width: (7 * $cell-width) + $padding-calendar * 2 + $cell-gap * 6;
```

Comment:
You can transfer mathematical operations to separate variables and use the result immediately, this will increase readability.